src/universal/styles/ui.js

```diff
@@ -360,7 +355,7 @@ const ui = {
   menuBackgroundColor: '#fff',
   menuBorderColor: appTheme.palette.mid30l,
   menuBorderRadius: borderRadiusSmall,
-  menuBoxShadow: shadow[2],
+  menuBoxShadow: shadow[3],
```

Comment:
I like the transition idea! At some point I think the shadow scale may be too great for certain components, like a menu having `shadow[4]`. I did add a dynamic prop to the `Button` component. If we add that prop to the card, we would have to pass it through the card footer to get it to those footer menus. Also, if it would have to be limited to the max number in our current scale (0-4) including any plus 1. So right now the focused card is `shadow[2]` and the menu is `shadow[3]`. The card could start at `shadow[0]` or `shadow[1]` ending in a menu at 3 or 4. If we don’t want to add more to the `shadow` array we could turn that into a function that returned a shadow at any depth based on the scale.

At this point I’d rather say this iteration is a good step forward, and revisit on another round. So your call if you want me to make the card > card footer > menu dynamic.